### DISCUSSION TITLE
DOM focus remains on trigger element

### DISCUSSION THREAD

**Discussion by jonathanbutler7** [floating-ui/floating-ui#1921] [React]
With [this example](https://codesandbox.io/s/gallant-sea-rcg43b?file=/src/Select.tsx), is there a way to leave the DOM focus on the trigger element, and indicate visual focus on the list items with virtual focus styles? I'm asking because of the spec screenshotted below from [w3](https://www.w3.org/WAI/ARIA/apg/example-index/combobox/combobox-autocomplete-none.html). 

**Disclaimer:** I also recognize that several other reputable libraries (like radix-ui, which uses floating-ui internally) move DOM focus (and also visual focus) to the list items, so if I'm misunderstanding the spec, please let me know.

<img width="968" alt="195927279-26a4344e-dde6-4a08-8cd4-ec09aead3bf7" src="https://user-images.githubusercontent.com/63125920/195927945-28559ec5-5d7f-4489-bc38-df3dca69b1ef.png">

**Comment by atomiks** [comment]
With a Select, you don't need focus to remain on the trigger unlike a Combobox, because the user does not type anything into the button. (They technically can, but it's not the main use case) 

The [`Autocomplete`](https://codesandbox.io/s/fragrant-water-bsuirj?file=/src/App.tsx) example demonstrates a combobox and "virtual focus".
